mattermost-plugin/server/configuration.go

```diff
@@ -71,21 +67,23 @@ func (p *Plugin) setConfiguration(configuration *configuration) {
 }
 
 // OnConfigurationChange is invoked when configuration changes may have been made.
-func (p *Plugin) OnConfigurationChange() error {
+func (p *Plugin) OnConfigurationChange() {
```

Comment:
Why is this no longer returning error? I think that would break the plugin interface itself.

Reply:
Yep. I fixed this in my upstream PR, but forgot to update here. Nice catch. It did break the interface. `GetUnsanitizedConfig` doesn't return an error, but `LoadPluginConfiguration` did.